controllers/navigationRoutes.js

```diff
@@ -157,8 +182,60 @@ router.get('/browse', async (req, res) => {
 router.use(helper.VerifyLoggedIn);
 
 // GET route for dashboard page (user profile/account)
-router.get('/dashboard', (req, res) => {
-  res.render('dashboard')
+router.get('/dashboard', async (req, res) => {
+  helper.SafeRequest(res, async (res) => {
+    const user = await User.findOne({
+      where: {
+        id: req.session.userID
+      },
+      attributes: ['userName']
+    })
+    const userFavorites = await UserRecipeFavorite.findAll({
+      where: {
+        userID: req.session.userID
+      },
+      include: {
+        model: Recipe
+      }
+    })
+   const userRecipeFav = userFavorites.map(obj => obj.get().Recipe.get())
+  
+   const userRecipes = await Recipe.findAll({
+      where: {
+        userID: req.session.userID
+      },
+   })
```

Comment:
still need to put these through the renderRecipe function so that recipe preview renders correctly